### ISSUE TITLE
Why DLLs are never built with static runtime?

### ISSUE THREAD

**Issue by dmitry-zakablukov** [conan-io/conan-package-tools#43]
In file `conan/packager.py`, method `_add_visual_builds()`:
```
        if shared_option_name:
            if "MT" in self.visual_runtimes:
                sets.append([{"build_type": "Release", "compiler.runtime": "MT"},
                             {shared_option_name: False}])
            if "MTd" in self.visual_runtimes:
                sets.append([{"build_type": "Debug", "compiler.runtime": "MTd"},
                             {shared_option_name: False}])
```
Why DLLs are never built with static runtime? It is very useful and I can't find any pros for the current behavior.

**Comment by lasote** [comment]
To be honest, I thought that it was not a very common use case, well to know.
Could you open a pull request? I could verify with some time if it works well with some of my packages but will require some time.
In any case, you can invoke the ``add_common_builds()`` and the add additional builds to ``builder.builds`` list.

**Comment by dmitry-zakablukov** [comment]
Pull request created: #44.

**Comment by lasote** [comment]
You can activate those builds using `dll_with_static_runtime=True` parameter to `add_common_builds`.
Available in 0.2.31. Thanks!

**Comment by dmitry-zakablukov** [comment]
Thank you, @lasote !
